src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -433,12 +440,24 @@ function EstimatedCallRow({
 
   const {group, isStartOfGroup, isEndOfGroup, isStartOfServiceJourney} =
     call.metadata;
+  const isFirstOfTrip = group === 'trip' && isStartOfGroup;
```

Comment:
Maybe this variable name could clarify what it is first of?

Reply:
Do you have a suggestion here? Would it be better with `isFirstOfTripGroup`? Kinda hard to find better naming without diverging from the group terms ('passed', 'trip', 'after') we have in the code.

Reply:
Yea I think `isFirstOfTripGroup` would be better :+1:
Maybe also `isStartOfTripGroup` to keep the naming in line with `isStartOfGroup`?